### PR TITLE
chore(bank_registry): updated bank_registry for CY, GR, IT, PT

### PR DESCRIPTION
**Pull request by dennisxtria**

### CODE REVIEW

File: schwifty/bank_registry/manual_gr.json [hunk 1]
```diff
@@ -174,6 +174,22 @@
         "bank_code": "057",
         "bic": "PRXBGRAA",
         "country_code": "GR"
+    },
+    {
+        "primary": true,
+        "name": "COOPERATIVE BANK OF CHANIA Cooperative of Limited Liabilities",
+        "short_name": "COOPERATIVE BANK OF CHANIA Cooperative of Limited Liabilities",
```
Comment (abourazanis) [inline schwifty/bank_registry/manual_gr.json:181]: that's doesnt look as a short name :P

File: schwifty/bank_registry/manual_gr.json [hunk 2]
```diff
@@ -174,6 +174,22 @@
...
+        "bank_code": "069",
+        "bic": "STXAGRA1",
+        "country_code": "GR"
+    },
+    {
+        "primary": true,
+        "name": "COOPERATIVE BANK OF KARDITSA",
+        "short_name": "COOPERATIVE BANK OF KARDITSA",
```
Comment (abourazanis) [inline schwifty/bank_registry/manual_gr.json:189]: 🚀 🚀

File: schwifty/bank_registry/manual_pt.json [hunk 3]
```diff
@@ -126,5 +126,45 @@
...
+      "primary": true,
+      "short_name": "CAIXA DE CRÉDITO AGRÍCOLA MÚTUO DE MAFRA, CRL"
+    },
+    {
+      "bank_code": "5340",
+      "bic": "CTIUPTPP",
+      "country_code": "PT",
+      "name": "CAIXA DE CRÉDITO AGRÍCOLA MÚTUO DE TORRES VEDRAS, C.R.L",
+      "primary": true,
+      "short_name": "CAIXA DE CRÉDITO AGRÍCOLA MÚTUO DE TORRES VEDRAS, C.R.L"
```
Comment (abourazanis) [inline schwifty/bank_registry/manual_pt.json:152]: their site is ccamtv, perhaps we can use this one as short name? still just an idea, ignore it freely

**Review by abourazanis — APPROVED** [review]

**Review by kziovas — APPROVED** [review]

File: schwifty/bank_registry/manual_pt.json (hunk 3 above)
Comment (dennisxtria) [reply schwifty/bank_registry/manual_pt.json:152]: I'd keep it the same and avoid any acronyms but if it ends up being the official name, we can rename then! 🤝

**Pull request merged**